From da3fb608311606a2fac57a969fde57ef93fca45e Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Thu, 5 Dec 2019 17:29:25 +0300
Subject: [PATCH 2/2] arm64: dts: r8a77980-v3hsk: Enable csi40 and csi41
 devices

This enables csi40 and csi41 devices which allows CSI UIO usage.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts b/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
index 5c8ded5..e72e8c4 100644
--- a/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
@@ -130,6 +130,14 @@
 	};
 };
 
+&csi40 {
+    status = "okay";
+};
+
+&csi41 {
+    status = "okay";
+};
+
 &du {
 	clocks = <&cpg CPG_MOD 724>,
 		 <&osc1_clk>;
-- 
2.7.4

